jmx_prometheus_javaagent/src/test/java/io/prometheus/jmx/JavaAgentIT.java

```diff
@@ -1,19 +1,16 @@
 package io.prometheus.jmx;
 
-import static org.hamcrest.MatcherAssert.assertThat;
+import org.hamcrest.CoreMatchers;
+import org.junit.Test;
 
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
+import java.io.*;
 import java.net.URL;
 import java.net.URLClassLoader;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.List;
+import java.util.*;
```

Comment:
As much as I'm not a fan of standalone, it should have equivalent functionality as far as is possible.